packages/slate-keymap/src/index.js

```diff
@@ -11,9 +11,16 @@ const Keymap = (shortcuts, options) => {
 
     const check = (event, editor) => isKeyPressed(event) && config.if(editor);
 
+    const handler =
+      typeof shortcuts[key] == "string"
+        ? (event, editor) => {
+            editor.command(shortcuts[key]);
```

Comment:
I wonder if it's more efficient to do `editor[shortcuts[key]]()`? As I understand it, the commands are also added to the editor as methods. 

Also, I would personally put `shortcuts[key]` in a const `arg` for legibility.

Reply:
Yes, they are also methods. I'm not sure on efficiency thought.

I agree on giving that a name. I went with `command`, since that's basically what it is, whether it's a function or a string.